Add Mineduc private subregistry for public schools: aprendemas.cl

huguei opened the pull request:
* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

Description of Organization
====

Mineduc is the Ministry of Education of the Chilean Government, responsible for promoting development at all levels of education at both public and private institutions.

Organization Website: https://www.mineduc.cl/

I work for NIC Chile (Hugo Salgado <hsalgado@nic.cl>), the .CL registry, submitting this PR on behalf of Mineduc. We're coordinated to validate the change. .CL whois doesn't list contact emails, but I can provide you with any proof at your request.

Reason for PSL Inclusion
====

Mineduc will use this private sub-registry to give each public school in Chile a website for e-learning infrastructure. Each site will be fully managed by school administrators, so they need independence for certificate issuing, cookie management, etc.


DNS Verification via dig
=======

Done:

```
$ dig _psl.aprendemas.cl txt +short 
"'https://github.com/publicsuffix/list/pull/1001'"

```

make test
=========

Tests were successfully run. No skips, fail or error.

dnsguru commented:
@huguei management at nic.cl also reached out directly about this being needed urgently for the shelter in place situation in Chile and the Ministry's needs.  I recall meeting you at one of the registry techdays where I presented the PSL to the registry operators during many of the ICANN meetings.

This in essence is adding an equivalence to 'edu.cl' as we might see in other zones.  

Thank you for taking the time to submit this.  In looking closer at the registry policies I'd like to encourage you to revise the overall .CL section to include your registration policies link, and it appears that .CL accepts general registrations at the second level, but the current entry does not indicate this.  That might also be a contributing factor.

sleevi commented:
Jothan: Did you submit the changes to the Wiki explaining how urgent
changes are not possible, and this will take 6-18 months to take effect in
practice?

sleevi commented:
> it appears that .CL accepts general registrations at the second level, but the current entry does not indicate this.

The current entry seems to reflect that just fine.

dnsguru commented:
Sort of.

In their case it appears that they don't have .CL itself listed.

https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion

I don't think I outright say 'there is no urgent'



Jothan Frakes
+1.206-355-0230 tel
+1.206-201-6881 fax


On Sat, Mar 21, 2020 at 9:37 AM sleevi <notifications@github.com> wrote:

> Jothan: Did you submit the changes to the Wiki explaining how urgent
> changes are not possible, and this will take 6-18 months to take effect in
> practice?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/publicsuffix/list/pull/1001#issuecomment-602069857>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACQTJL7KUQ76YRU5MC7IRLRITUOHANCNFSM4LQ4KQRA>
> .
>

dnsguru commented:
> > it appears that .CL accepts general registrations at the second level, but the current entry does not indicate this.
> 
> The current entry seems to reflect that just fine.

My mobile client didn't indicate this but I see it is the case as I sit down at my desk on this lovely Saturday.

dnsguru commented:
I have contacted the submitter with a request to add the DNS entry and to revise the .CL section they are submitting to align with the formatting used in other ccTLDs

sleevi commented:
Yeah, we should probably add a clarification that incorporation is O(months) to O(years). Given the use case and need for this, the most effective immediate solution is using domains under an existing suffix, or at least hosting the separable content under such names.

Not opposed to adding this name (without the header), but especially if you’re in contact with the registry beyond this issue, we should highlight that this won’t provide the resolution desired.

dnsguru commented:
I am embarrassed but I coughed while copying and pasting the CL section out of the PSL and missed the CL by itself.  Best I just own up on that.

Anyway, on the concept of URGENT and timing expectations, the trickle out effect of other implementation providers/people use or incorporate the PSL is beyond our control or ability to expedite.  Unfortunately that is where people seem to experience the circumstances that motivate the changes.

huguei commented:
Thank you guys. I appreciate your help on this. We're aware of the different parts that we'll gonna need to have it working, we'll go step by step!
I've just updated the PR for sorting the rest of names under CL, and added more clarification texts. Hope it's enough!

huguei commented:
TXT verification token already set in the zone.

dnsguru commented:
Confirmed _psl.aprendemas.cl txt record at centralops and 1.1.1.1

On Sat, Mar 21, 2020, 11:37 AM Hugo Salgado <notifications@github.com>
wrote:

> TXT verification token already set in the zone.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/publicsuffix/list/pull/1001#issuecomment-602084862>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACQTJNIRWSPEAC4AKCL4HTRIUCPJANCNFSM4LQ4KQRA>
> .
>

huguei commented:
> Could you remove the descriptions for the individual domains (e.g. “Chilean government subregistries”) and their approval? We try to only do that for the effective scope of the domain boundary (in this case, the cl and all that nic.cl operates)

Changes applied.

sleevi commented:
As noted, it can take quite a long time before client software reflects this.

1) Do not get a wildcard certificate for this domain, as when clients update, they will prohibit such wildcards (same as how they would prevent for `*.cl` or `*.com`)
2) Be aware that until clients upgrade, all storage (cookies, quota, etc) will be shared. Any school will be able to affect any other school.

huguei commented:
Thanks a lot @sleevi and @dnsguru for your help with this! We'll be careful with certs and cookies until it's safe.
